datahub-frontend/app/auth/sso/oidc/custom/CustomOidcAuthenticator.java

```diff
@@ -83,13 +87,13 @@ public CustomOidcAuthenticator(final OidcConfiguration configuration, final Oidc
           chosenMethod);
     }
 
-    final ClientID _clientID = new ClientID(configuration.getClientId());
+    final ClientID clientID = new ClientID(configuration.getClientId());
```

Comment:
nice :')

Reply:
Yeah, checkStyle apparently wasn't running on any of these files before and it started to after changing out the plugin 😆